### ISSUE TITLE
clicking on the repo <span> on the front page takes one to the external site

### ISSUE THREAD

**Issue by mdaniel** [sourcebot-dev/sourcebot#31]
I guess whether this is a bug, improvement, or `WONTFIX` depends on whether one thinks of sourcebot as merely a "fancy bookmarking" system, and the end user really wants to navigate to the code hosting site, or whether one thinks of sourcebot as "references as a service" and thus clicks should mostly navigate to other sourcebot URLs unless otherwise designated

1. Navigate to https://demo.sourcebot.dev/
2. Observe the `commaai/openpilot` in the _(ahem)_ horizontal scroll list
3. Click on it, expecting to be taken to https://demo.sourcebot.dev/search?query=repo%3Acommaai%2Fopenpilot
4. 😢  Instead observe that one is taken to https://github.com/commaai/openpilot

**Comment by KonradStanski** [comment]
I'm in the "sourcebot should aim to be a fully self sufficient tool aiming to reference to itself as much as possible" camp.
With this, I feel that when clicking on the repo carousel, this should take the user to the as of yet not implemented file browser for a repo.
For now it is fine.

**Comment by brendan-kellam** [comment]
Agreed with @KonradStanski - once we have a file explorer, we can link to it from other points in the app. #94

**Comment by brendan-kellam** [comment]
Fixed in #528
